common/djangoapps/student/tasks.py

```diff
@@ -0,0 +1,89 @@
+from celery import shared_task
+from django.conf import settings
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from common.djangoapps.course_modes.models import CourseMode
+from common.djangoapps.student.helpers import get_course_dates_for_email, get_instructors
+from lms.djangoapps.utils import get_braze_client
+from openedx.core.djangoapps.catalog.utils import get_course_uuid_for_course, get_owners_for_course, \
+    get_course_run_details
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.features.course_experience import ENABLE_COURSE_GOALS
+
+
+@shared_task()
+def course_enrollment_email(user_id, course_id, request=None):
+    """
+        Retrieve information about course for sending enrollment email
+    """
+
+    course_run_fields = [
+        'key', 'title', 'short_description', 'marketing_url', 'pacing_type', 'min_effort',
+        'max_effort', 'weeks_to_complete', 'enrollment_count', 'image', 'staff',
+    ]
+
+    try:
+        user = User.objects.get(id=user_id)
+    except User.DoesNotExist:
+        user = None
+
+    try:
+        course_key = CourseKey.from_string(course_id)
+    except InvalidKeyError:
+        course_key = None
+
+    trigger_properties = {
+        'instructors': [],
+        'course_date_blocks': [],
+        'learners_count': '',
+        'partner_image_url': '',
+        'banner_image_url': ''
+    }
+
+    course_dates_list = get_course_dates_for_email(user, course_key, request)
+    course_uuid = get_course_uuid_for_course(course_id)
+    owners = get_owners_for_course(course_uuid=course_uuid)
+    course_run = get_course_run_details(course_id, course_run_fields)
+
+    if course_run:
+        trigger_properties = {}
+        marketing_root_url = settings.MKTG_URLS.get('ROOT')
+        instructors = get_instructors(course_run, marketing_root_url)
+        trigger_properties.update({
+            'instructors': instructors,
+            'instructors_count': 'even' if len(instructors) % 2 == 0 else 'odd',
+            'pacing_type': course_run.get('pacing_type'),
+            'min_effort': course_run.get('min_effort'),
+            'max_effort': course_run.get('max_effort'),
+            'weeks_to_complete': course_run.get('weeks_to_complete'),
+            'learners_count': '{:,}'.format(course_run.get('enrollment_count')),
+            'course_title': course_run.get('title'),
+            'short_description': course_run.get('short_description'),
+            'banner_image_url': course_run.get('image').get('src') if course_run.get('image') else ''
```

Comment:
Also you miss to move marketing url
`'marketing_url': course_run.get('marketing_url'),`

Reply:
Marketing URL is not in Ticket's comment.